examples/experimental/dagster-airlift/dagster_airlift/core/sensor.py

```diff
@@ -210,6 +213,7 @@ def get_dag_run_metadata(dag_run: DagRun) -> Mapping[str, Any]:
         "Run Details": MarkdownMetadataValue(f"[View Run]({dag_run.url})"),
         "Start Date": TimestampMetadataValue(dag_run.start_date),
         "End Date": TimestampMetadataValue(dag_run.end_date),
```

Comment:
To be clear this separates out `EFFECTIVE_TIMESTAMP_METADATA_KEY` (which we will programmatically use) from `"End Date"` which is just for display?

Reply:
Btw making all these hardcoded strings constants would be a nice thing to do

Reply:
That's right yea